wsd/ClientRequestDispatcher.cpp

```diff
@@ -406,13 +406,28 @@ class ConvertToAddressResolver : public std::enable_shared_from_this<ConvertToAd
     std::vector<std::string> _addressesToResolve;
     ClientRequestDispatcher::AsyncFn _asyncCb;
     bool _allow;
+    bool _capabilityQuery;
+
+    void logAddressIsDenied(const std::string& addressToCheck) const
+    {
+        // capability queries if convert-to is available in order to put that
+        // info in its results. If disallowed this isn't an attempt by an
+        // unauthorized host to use convert-to, only a query to report if it is
+        // possible to use convert-to.
+        if (_capabilityQuery)
+            LOG_INF_S("convert-to: Requesting address is denied: " << addressToCheck);
+        else
+            LOG_WRN_S("convert-to: Requesting address is denied: " << addressToCheck);
```

Comment:
These don't need to be `_S`.

Also, do we need to log at info level that a given IP doesn't have convert-to caps? Perhaps this is more of a debug thing?

Reply:
https://github.com/CollaboraOnline/online/pull/11994 for that follow up